feature-wallet-impl/src/main/java/jp/co/soramitsu/feature_wallet_impl/data/mappers/OperationMappers.kt

```diff
@@ -10,33 +9,37 @@ import jp.co.soramitsu.feature_wallet_api.domain.model.WalletAccount
 import jp.co.soramitsu.feature_wallet_api.domain.model.amountFromPlanks
 import jp.co.soramitsu.feature_wallet_impl.data.network.model.response.SubqueryHistoryElementResponse
 import jp.co.soramitsu.feature_wallet_impl.presentation.model.OperationModel
+import kotlin.time.ExperimentalTime
+import kotlin.time.seconds
 
 fun mapOperationStatusToOperationLocalStatus(status: Operation.Status) = when (status) {
     Operation.Status.PENDING -> OperationLocal.Status.PENDING
     Operation.Status.COMPLETED -> OperationLocal.Status.COMPLETED
     Operation.Status.FAILED -> OperationLocal.Status.FAILED
 }
 
+@ExperimentalTime
 fun mapOperationToOperationLocalDb(operation: Operation, source: OperationLocal.Source): OperationLocal {
     with(operation) {
         val operationLocal = OperationLocal(
             hash = hash,
             address = address,
-            time = time.secondsToMilliseconds(),
+            time = time.seconds.toLongMilliseconds(),
             tokenType = mapTokenTypeToTokenTypeLocal(tokenType),
-            type = transactionType.header,
-            call = transactionType.subheader,
+            type = transactionType.getHeader(),
+            call = transactionType.getSubheader(),
             amount = transactionType.operationAmount.toBigInteger(),
             fee = transactionType.operationFee.toBigInteger(),
             status = mapOperationStatusToOperationLocalStatus(operation.transactionType.status),
-            source = source
+            source = source,
+            operationType = transactionType.getOperationType()
         )
 
         return when (transactionType) {
```

Comment:
`when(val type = transactionType) `

then type will be smartcased in inside `is X -> {}` expressions